2018/python/file_generator.py

```diff
@@ -351,7 +352,7 @@ def validate_quality_edit_file(self, quality_filename):
 			res_df = pd.DataFrame(checker.results)
 			new_df = res_df[(res_df['status']=='failed')]
 
-			#If there are no syntax or validity edits,
+			#If there are no syntax or validity edits or row_ids is not in column names,
```

Comment:
Can you rephrase this so it is more accessible?

Reply:
Sure! Some edits that are not row specific generate an edit report without row_ids as a column. These edits were not being generated correctly, and so I decided to exclude them as checks in the validation function here. 

However, I think I need to resolve the issue with this PR here.